csuntechlab/iSTART
# STR-232 -- FE: Toggle quiz selection

# Description
Made quiz questions toggle-able.
  
# Testing
1. Ensure quiz options can be toggled.
2. Check to make sure when un-toggled, you are able to continue.
3. Make sure that each toggle does not remove other option toggle.
